bindings/wysiwyg-ffi/src/ffi_composer_model.rs

```diff
@@ -27,6 +27,24 @@ impl ComposerModel {
         ))
     }
 
+    pub fn replace_all_text(
```

Comment:
I think I would prefer the name `replace_all_markdown`. Was there a reason you went for "text"?